README.md

```diff
@@ -178,7 +181,7 @@ Options are:
 - `darwin` (Mac OSX)
 - `win32` (Windows)
 - `linux` (Linux)
-
+- `all` (Mac OSX + Windows + Linux)
```

Comment:
I would love to see it as 
```
`all` (default, Mac OSX + Windows + Linux)
```
or simply
```
`all` (default)
```
as the context should render the selection clear.